sdk/typescript/.changes/unreleased/Breaking-20240827-015828.yaml

```diff
@@ -0,0 +1,9 @@
+kind: Breaking
+body: |-
+  Drop CommonJS support in favor of module to support new `graphql-request` version.
+
+  Bump `Bun` runtime version to `1.1.26` to support new `execa` version.
```

Comment:
I feel like I've asked this question many times now - are there modules in the daggerverse that will break because of this? Do we have any idea of the impact of this? Is common js support something that users are likely to be relying on?

Introducing this breaking change without module compatibility worries me. If we are definitely doing that, we should probably write that in the change log - since ideally, *completely* breaking changes with no compatibility layer provided should be the exception instead of the default.

Reply:
Yeah I'm looking at the TS module list, will do an update once I have a clearer view.